### PR TITLE
List of States in Pakistan

### PR DESCRIPTION
**Pull request by AnwerAR**
Added PK.php in states for Pakistani states. all province codes are accoring to ISO 3166 Standard

### CODE REVIEW

**Comment by mikejolley** [comment]
Should these be alphabetically listed?

**Comment by AnwerAR** [comment]
Yes, its okay to show in alphabetic order. because i have no other sorting order or priority for any state.

File: i18n/states/PK.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Pakistan's states
+ *
+ * @author      WooThemes
+ * @category    i18n
+ * @package     WooCommerce/i18n
+ * @version     2.0.0
```
Comment (justinshreve) [inline i18n/states/PK.php]: This should be the current version the change was introduced in, so 2.7.0.

File: i18n/states/PK.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Pakistan's states
+ *
+ * @author      WooThemes
```
Comment (justinshreve) [inline i18n/states/PK.php]: This should be WooCommerce now.

**Review by emdashcodes — COMMENTED** [review]
In addition to updating the sorting, I have 2 little comments on the docblock as well.

File: i18n/states/PK.php (hunk 1 above)
Comment (AnwerAR) [inline i18n/states/PK.php]: Thank You for your guidance @justinshreve . i am contributing to WooCommerce for the first time.

**Pull request merged**